src/wallet/rpcwallet.cpp

```diff
@@ -2984,7 +2984,7 @@ UniValue z_listaddresses(const UniValue& params, bool fHelp)
     return ret;
 }
 
-CAmount getBalanceTaddr(std::string transparentAddress, int minDepth=1) {
```

Comment:
should we add an option to see watch only addresses in `z_listaddresses`?

Reply:
:+1:, I would love to be able to differentiate watch-only addresses in ZEC+HUSH

Reply:
@str4d Worth filing a ticket about this.  Curently, it's not obvious when calling `z_listadresses` which ones are watch-only.  Perhaps future json output should be something like `{"addresses":[...], "watchonly":[ ... ]}`